docs/features.md

```diff
@@ -36,3 +35,24 @@ protected function isTrue(): Attribute
 }
 ```
 
+## Model Relationships
+
+In order for Larastan to recognize Model relationships:
+- the return type must be defined
+- the method must be `public`
+- the class argument must be a literal string (not a variable)
+- adding the docblock `@return` helps
```

Comment:
No. The `@return` tag is need only when the above criteria is not met.

Reply:
I said it helps...I just always add the docblocs on all my relationships but I can update

Reply:
updated